### ISSUE TITLE
CouchDB v. 3.0.0 cannot create admin on macOS Catalina 10.15.3

### ISSUE THREAD

**Issue by CharaLagou** [apache/couchdb#2705]
CouchDB v. 3.0.0 cannot create admin on macOS Catalina 10.15.3

## Description

CouchDB 3.0.0 installed from installer. 
Cannot create admin. 

## Steps to Reproduce

1. Install CouchDB from [installer](https://neighbourhood.ie/download-apache-couchdb-mac/)
2. Double click zipped file
3. Drag and drop the Apache CouchDB.app into Applications folder
4. Open Apache CouchDB.app 

This automatically opens a window at [login page](http://127.0.0.1:5984/_utils/#login).
Databases, setup, active tasks, replication and documentation pages are not accessible and the app redirects to login page.
Configuration and verification are accessible. When trying to verify the installation the following error messages appear: 
"Error: Database does not exist."
"Error: Cannot read property 'set' of undefined"
"Error: You are not a server admin."

Since there is no admin, I attempted to create one visiting http://127.0.0.1:5984/_utils/#createAdmin/couchdb@127.0.0.1 but it didn't succeed. 

<img width="1431" alt="Screenshot 2020-03-23 at 00 04 06" src="https://user-images.githubusercontent.com/11322049/77265258-8e84fe80-6c9c-11ea-8467-cddca61dcd3a.png">

## Expected Behaviour

Successful creation of admin.

## Your Environment

* CouchDB version used: 3.0.0
* Browser name and version: same result on Chrome, Safari
* Operating system and version: macOS Catalina 10.15.3

**Comment by janl** [comment]
can you share your couch log, your default and local ini, you're vm.args and a full listing of your data directory?

**Comment by CharaLagou** [comment]
@janl thank you very much for your response. 
I have good news!
While I was trying to find the files you asked for I realised that in the folder User/Library/Application Support/CouchDB2 there were files from a previous installation,  based on their date of creation. I deleted the whole folder and reinstalled CouchDB 3.0.0 using the installer and this time I immediately got a window to setup an admin when I opened the app. Installation verification was successful too. 
Thanks for your time and consideration.
